shared/tasks/run-bats.sh

```diff
@@ -31,6 +31,12 @@ chmod 600 $ssh_key_path
 export BOSH_GW_PRIVATE_KEY=$ssh_key_path
 
 pushd bats
+  # mock: alicloud does not support disk size less than 20GB
+  if [ ${INFRASTRUCTURE} == "alicloud" ];
+  then
+    sed -ig "s/2048/20480/" ./spec/system/with_release_stemcell_spec.rb
+    sed -ig "s/4096/40960/" ./spec/system/with_release_stemcell_spec.rb
```

Comment:
@demonwy this is fragile. Does the CPI automatically upsize disks which are smaller than the IaaS minimum?

@cdutra would it be acceptable/normal for a CPI to automatically give a larger disk if the IaaS doesn't support the smaller, user-requested size?

Reply:
@dpb587-pivotal  thanks for your advice. 
I think CPI cant not change disk size without the consent of the user.
@cdutra  any suggestion?

Reply:
@dpb587-pivotal @demonwy If the IaaS doesn't support that specific size it should fail if specifying smaller than acceptable size. Also it should be documented in the cpi docs on bosh.io